en/identity-server/next/docs/apis/organization-apis/restapis/organization-user-share.yaml

```
@@ -384,7 +391,9 @@ paths:
         - User Accessible Roles
       summary: Get roles assigned to a user in an organization
       description: |
-        This API fetches the roles assigned to the specified user within a particular organization, with support for pagination, filtering, and recursion.
+        This API fetches the roles assigned to the specified user within a particular organization. Supports pagination, filtering, and recursion. </br>
```

```suggestion
        This API fetches the roles assigned to the specified user within a particular organization. </br>
```
Let's remove pagination support.

---

Verified.